source/layouts/layout.html.slim

```diff
@@ -14,4 +14,6 @@ html
 		== favicon_tag "apple-touch-icon-120x120.png", href: image_path("apple-touch-icon-120x120.png"), rel: 'apple-touch-icon', type: 'image/png', sizes: "120x120"
 		== favicon_tag "apple-touch-icon-152x152.png", href: image_path("apple-touch-icon-152x152.png"), rel: 'apple-touch-icon', type: 'image/png', sizes: "152x152"
 	body[ng-app="BitriseWorkflowEditor" ng-cloak]
+		- if build?
```

Comment:
This is a bit confusing, should it differentiate between local and website mode?